src/test/regress/citus_tests/common.py

```diff
@@ -93,7 +93,7 @@ def get_pg_major_version():
     14: "10.2.0",
     15: "11.1.5",
     16: "12.1.5",
-    17: "13.0.0",
+    17: "13.0.1",
```

Comment:
I'm not sure if we update this function at every patch release but this looks okay